backend/app/services/api/sorter.rb

```diff
@@ -0,0 +1,61 @@
+module API
+  class Sorter
+    attr_accessor :query, :sorting_query_param, :sorting_direction_param, :language
+
+    SORTING_DIRECTIONS = %i[asc desc]
+    SORTING_OPTIONS = {
+      name: {
+        localized: true,
+        specific: {ProjectDeveloper => {relation: :account, localized: false},
```

Comment:
We can check which columns have translations and configure filters accordingly, for example `Investor.locale_columns`

Reply:
I would prefer to do it automatically to not forget about it.